docs/interaction_code_flow.md

```diff
@@ -600,10 +600,10 @@ Enable or disable widget functionality with the following options.
   Pre-fills the identifier field with the previously used username.
 
 ### Flow
-By default, Widget will either proceed with an existing flow or start a new authenticate flow. This option allows bootstrapping the Widget into a specific view such as register or forgot password. Using this option, it is possible to build specific pages for login, register, and forgot password.
+The Okta Sign-In Widget will either proceed with a current flow or start a new authenticate flow by default. This option allows bootstrapping the widget into a specific view such as register, unlock, or forgot password. The widget can be used in particular pages such as login, forgot-password, and unlock pages with the appropriate value set in the `flow` config option  
```

Comment:
nit: Should we change "forgot password" to "reset password" since that's the flow name?